src/Associations.cc

```diff
@@ -48,475 +49,473 @@ Associations::Associations()
+        s.vx = sqr(0.1 / 3600 / cos(coord.getLatitude()));
+        s.vy = sqr(0.1 / 3600);
+        s.vxy = 0.;
+        RefStar *r = new RefStar(s, s);
+        refStarList.push_back(r);
     }
 
-  // project on CTP (i.e. RaDec2CTP), in degrees
-  GtransfoLin identity;
-  TanRaDec2Pix RaDec2CTP(identity, commonTangentPoint);
+    // project on CTP (i.e. RaDec2CTP), in degrees
+    GtransfoLin identity;
+    TanRaDec2Pix RaDec2CTP(identity, commonTangentPoint);
 
-  AssociateRefStars(usnoMatchCut, &RaDec2CTP);
+    AssociateRefStars(usnoMatchCut, &RaDec2CTP);
 }
 
 const lsst::afw::geom::Box2D Associations::GetRaDecBBox()
 {
-  // compute the frame on the CTP that contains all input images
-  Frame tangentPlaneFrame;
+    // compute the frame on the CTP that contains all input images
+    Frame tangentPlaneFrame;
 
-  for (CcdImageIterator i=ccdImageList.begin(); i!= ccdImageList.end(); ++i)
+    for (CcdImageIterator i = ccdImageList.begin(); i != ccdImageList.end(); ++i)
     {
-      CcdImage &ccdImage = **i;
-      Frame CTPFrame = ApplyTransfo(ccdImage.ImageFrame(),*ccdImage.Pix2CommonTangentPlane(),LargeFrame);
-      if (tangentPlaneFrame.Area() == 0) tangentPlaneFrame = CTPFrame;
-      else tangentPlaneFrame += CTPFrame;
+        CcdImage &ccdImage = **i;
+        Frame CTPFrame = ApplyTransfo(ccdImage.ImageFrame(), *ccdImage.Pix2CommonTangentPlane(), LargeFrame);
+        if (tangentPlaneFrame.Area() == 0) tangentPlaneFrame = CTPFrame;
+        else tangentPlaneFrame += CTPFrame;
     }
 
-  // convert tangent plane coordinates to RaDec:
-  GtransfoLin identity;
-  TanPix2RaDec CTP2RaDec(identity, commonTangentPoint);
-  Frame raDecFrame = ApplyTransfo(tangentPlaneFrame,CTP2RaDec,LargeFrame);
+    // convert tangent plane coordinates to RaDec:
+    GtransfoLin identity;
+    TanPix2RaDec CTP2RaDec(identity, commonTangentPoint);
+    Frame raDecFrame = ApplyTransfo(tangentPlaneFrame, CTP2RaDec, LargeFrame);
 
-  lsst::afw::geom::Point<double> min(raDecFrame.xMin, raDecFrame.yMin);
-  lsst::afw::geom::Point<double> max(raDecFrame.xMax, raDecFrame.yMax);
-  lsst::afw::geom::Box2D box(min, max);
+    lsst::afw::geom::Point<double> min(raDecFrame.xMin, raDecFrame.yMin);
+    lsst::afw::geom::Point<double> max(raDecFrame.xMax, raDecFrame.yMax);
+    lsst::afw::geom::Box2D box(min, max);
 
-  return box;
+    return box;
 
 }
 
-void Associations::AssociateRefStars(const double &MatchCutInArcSec,
-				     const Gtransfo* T)
+void Associations::AssociateRefStars(double MatchCutInArcSec, const Gtransfo* T)
 {
-  //DEBUG
-  //  std::cout << "****" << std::endl << refStarList << std::endl;
-  // std::cout << "fitted list size " << fittedStarList.size() << endl;
-  // std::cout << fittedStarList << std::endl;
+    //DEBUG
+    //  std::cout << "****" << std::endl << refStarList << std::endl;
+    // std::cout << "fitted list size " << fittedStarList.size() << endl;
+    // std::cout << fittedStarList << std::endl;
```

Comment:
I'm not going to remove any of the commented code in jointcal until I have a better understanding of where it comes from/whether it might be useful. That'll come with time, so this stuff will stay for now.